apps/dashboard/app/models/files.rb

```diff
@@ -32,10 +32,10 @@ def self.stat(path)
       id: "dev-#{s.dev}-inode-#{s.ino}",
       name: path.basename,
       size: s.directory? ? 'dir' : s.size,
+      human_size: ::ApplicationController.helpers.number_to_human_size(s.size, precision: 3),
       directory: s.directory?,
-      date: s.mtime.strftime("%d.%m.%Y"),
+      date: s.mtime.utc.iso8601,
```

Comment:
First we convert s.mtime (Time) to a string

    date: s.mtime.utc.iso8601,
    
Then we convert this string to an integer (num milliseconds) and finally to a JavaScript Date object:

    let date = new Date(Date.parse(data))

Why not stick with integers?

    # server side convert to seconds
    date: s.mtime.to_i
    
    # client side convert to ms
    let date = new Date(data*1000)


If you are going to first convert the modified_at date to number of ms from epoc `Date.parse(data)`